ci-operator/step-registry/hypershift/mce/kubevirt/create/disconnected/hypershift-mce-kubevirt-create-disconnected-chain.yaml

```diff
@@ -0,0 +1,208 @@
+chain:
+  as: hypershift-mce-kubevirt-create-disconnected
+  steps:
+    - as: create-hostedcluster
+      cli: latest
+      env:
+        - name: HYPERSHIFT_NODE_COUNT
+          default: "3"
+          documentation: "The number nodes to automatically create and join to the cluster."
+        - name: ENABLE_ICSP
+          default: "false"
+          documentation: "if true, add image content sources config(path=${SHARED_DIR}/mgmt_iscp.yaml)"
+      commands: |-
+        set -exuo pipefail
+
+        source "${SHARED_DIR}/packet-conf.sh"
```

Comment:
This script is big enough that I think we should place it in a file, `ci-operator/step-registry/hypershift/mce/kubevirt/create/disconnected/hypershift-mce-kubevirt-create-disconnected-commands.sh`